spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -247,6 +255,7 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			isDownStreamAppPartitioned = isPartitionedConsumer(appDeployTimeProperties,
 					upstreamAppSupportsPartition);
 
+			logger.info(String.format("Downloading resource URI [%s]", registration.getUri()));
 			Resource appResource = registration.getResource();
 			Resource metadataResource = registration.getMetadataResource();
```

Comment:
Yes, maybe at debug level since the main issue was that it appeared that the server was not responding to a deployment request since the time to download an application jar can be large due to the size of the app.  Metadata jars are much smaller.    

To be consistent we should also indicate we are downloading the metadata when doing `app info` or for tab completion. Not need for 1.2 RC, but should create an issue to get to it another time.